### PR TITLE
HDDS-8648. Improving OmKeyInfo.Builder and OmDirectoryInfo.Builder usage

### PR DESCRIPTION
**Pull request by VarshaRaviCV**
## What changes were proposed in this pull request?

Improving the builder usage of OmKeyInfo and OmDirectoryInfo

## What is the link to the Apache JIRA

https://issues.apache.org/jira/browse/HDDS-8648

## How was this patch tested?

Unit tests

### CODE REVIEW

File: hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/file/OMFileRequest.java [hunk 1]
```diff
@@ -707,21 +707,21 @@ public static OmKeyInfo getOmKeyInfo(String volumeName, String bucketName,
...
-    builder.setModificationTime(dirInfo.getModificationTime());
-    builder.setObjectID(dirInfo.getObjectID());
-    builder.setUpdateID(dirInfo.getUpdateID());
-    builder.setFileName(dirInfo.getName());
-    builder.setReplicationConfig(RatisReplicationConfig
-        .getInstance(HddsProtos.ReplicationFactor.ONE));
-    builder.setOmKeyLocationInfos(Collections.singletonList(
-            new OmKeyLocationInfoGroup(0, new ArrayList<>())));
+    builder.setParentObjectID(dirInfo.getParentObjectID())
+            .setKeyName(keyName)
```
Comment (sodonnel) [inline hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/file/OMFileRequest.java:711]: These indents look like 8 spaces. They should be 4 to keep with the project style.

Comment (sodonnel) [reply hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/file/OMFileRequest.java:711]: Also I think you can simpliy this furher like:

```
return new OmKeyInfo.Builder()
    .setXyz()
    .setabc()
    .build();
```

Comment (VarshaRaviCV) [reply hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/file/OMFileRequest.java]: Fixed.

**Review by sodonnel — APPROVED** [review]
LGTM pending green CI.

**Pull request merged**